Documentation/Signoffs/launcher-aiming-signoff.md

```diff
@@ -0,0 +1,59 @@
+## Constraints 
+* C1: Create an interceptor capable of functioning
+on its own without outside interaction.
+* C3: Create an interceptor capable of fitting inside
+a 1x1x1 foot box before powering on.
+* C20: Launcher pitch and yaw must be controllable to a precision of within 0.6 degrees
+* C21: Time to move launcher into aiming position must be less than 0.40 seconds.
+* Aiming speed will be calculated from worst-case torque requirements provided by the Mechanical Engineering group
+## Explainations for C20 and C21
+For C20, the precision of the actuators needs to be taken into account because the less precise that the angle of the launcher is, the less accurate the intercepting shot will be over a distance. In addition, the precision is crucial because the intercepting projectile will have a dispersion from the target associated with it. To find the dispersion of the intercepting projectile, an experiment was conducted to find worst-case dispersion of the intercepting projectile. The mechanical engineering group have specified that the intercepting projectile will be a 3-inch long, half inch diameter cylinder and the projectile exit velocity will be around 30 feet per second. It was also specified that the maximum distance that the the launcher would attempt to intercept a projectile would be around 36 inches from the back of the blue square of the game board. Given that it would be difficult to build an exact design of the launcher at this time, the expiriment uses a nerf dart gun which has a similar projectile size, shape and exit velocity as what was specified by the mechanical engineering group. To control the projectile spread, the gun was mounted in such a way that the position of the gun would not change from shot to shot. Once the gun was mounted, a target was set up 36 inches away given that would be the farthest desired distance to intercept. To keep track of where the dart hit the target, liquid chalk was applied to the dart so as it impacted it would show a mark indicating where it had hit the target. After firing 10 shots with this setup, it was determined that the maximum dispersion between shots was about 1.25 inches in diameter. It was then inferred that to noticably change the direction of the target, the center of the dart would have to hit the outer edge of the target. Factoring in the maximum dispersion diameter of 1.25 inches from the expirement, it was calculated that the maximum tolerance that the interceptor could have is +/- 0.74 degrees from 36 inches away. To be conservative the constraint will be 20 percent tighter allowing for around +/- 0.6 degrees of precision of the pitch and yaw actuators. 

```

Comment:
There's no reason to have this much text. 1 use paragraphs. 2 is said much more eloquently in a picture or in math.

Reply:
I told chatGPT to get rid of unnecessary detail. This is what it gave back:

In C20, actuator precision is crucial for accurate intercepting shots. An experiment using a nerf dart gun, similar to the specified projectile, found a worst-case dispersion of 1.25 inches. To reliably change the target's direction, the interceptor's tolerance must be within +/- 0.74 degrees from 36 inches away. To ensure reliability, a 20 percent tighter constraint is set, allowing for around +/- 0.6 degrees of precision for pitch and yaw actuators.

Reply:
My question is, how did you arrive at 0.74 degrees. Don't explain in more words. Use a picture to explain your reasoning. Hand drawn and uploaded is fine if that is easier.